src/quick-lint-js/parse.h

```diff
@@ -1290,12 +1287,32 @@ class parser {
       QLJS_PARSER_UNIMPLEMENTED();
```

Comment:
Nit: The comment on line 1283 doesn't make sense anymore. (`return` is no different from `break`.)

See also my comment on test_parse.function_statement_without_parameter_list_or_body.